examples/Demo/Shared/Pages/KeyCode/Examples/KeyCodeGlobalExample.razor

```diff
@@ -9,29 +14,36 @@
 
     @foreach (var key in KeyPressed)
     {
-        <span class="key">@key</span>
+        <span class="key" type="@key.Event">@key.Key</span>
     }
 </FluentCard>
 
 @code
 {
-    private List<string> KeyPressed = new();
+    private bool IncludeKeyUp = false;
+    private List<(string Key, string Event)> KeyPressed = new();
 
     protected override void OnInitialized()
     {
-        KeyCodeService.RegisterListener(OnKeyDownAsync);
+        // FluentKeyCode.PreventMultipleKeyDown = true;
+        KeyCodeService.RegisterListener(OnKeyDownAsync, OnKeyDownAsync);
```

Comment:
I understand from the PR description there is now the option to supply both KeyDown and KeyUp handler to the service. But naming them both `OnKeyDownAsync` here looks a bit weird. Perhaps rename the method to `OnKeyHandleAsync`?